cobc/codegen.c

```diff
@@ -10229,7 +10246,8 @@ output_report_define_lines (int top, struct cb_field *f, struct cb_report *r)
 	if ((f->report_flag & COB_REPORT_LINE)
```

Comment:
(originating lines 10229-10235) This _should_ be moved to typeck.c, _at least_ the warning.